test/test_nn.py

```diff
@@ -3155,6 +3155,9 @@ def func(x):
         gradcheck(func, [x])
         gradgradcheck(func, [x])
 
+        x = torch.randn(2, 2, 2)
+        self.assertRaises(RuntimeError, lambda: F.fractional_max_pool2d(x, (1, 1), output_size=(3, 3)))
```

Comment:
Hmmm, does this actually successfully trigger the error message? You have `output_size = (3, 3)` here, which are `> 1`.